staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -449,6 +454,25 @@ const (
 	SupplementalGroupsStrategyRunAsAny SupplementalGroupsStrategyType = "RunAsAny"
 )
 
+// RuntimeClassStrategyOptions define the strategy that will dictate the allowable RuntimeClasses
+// for a pod.
+type RuntimeClassStrategyOptions struct {
+	// AllowedRuntimeClassNames is a whitelist of RuntimeClass names that may be specified on a pod.
+	// A value of "*" means that any RuntimeClass name is allowed, and must be the only item in the
+	// list. An empty list requires the RuntimeClassName field to be unset.
+	AllowedRuntimeClassNames []string `json:"allowedRuntimeClassNames" protobuf:"bytes,1,rep,name=allowedRuntimeClassNames"`
+	// DefaultRuntimeClassName is the default RuntimeClassName to set on the pod.
+	// The default MUST be allowed by the above AllowedRuntimeClasses.
```

Comment:
Hmm
```
The default MUST be allowed by the above AllowedRuntimeClasses
```

should be
```
The default MUST be allowed by the above AllowedRuntimeClassNames
```

Right? (i think @liggitt said the same thing above?)

Reply:
Fixed